yarn-project/noir-bb-bench/src/index.ts

```diff
@@ -1,77 +1,62 @@
 /* eslint-disable camelcase */
-import { type ForeignCallOutput, Noir } from '@aztec/noir-noir_js';
+import { Noir } from '@aztec/noir-noir_js';
 import type { InputValue } from '@aztec/noir-noirc_abi';
 
 import createDebug from 'debug';
 
 // these files are generated
 import Circuit1 from '../artifacts/circuit_1.json' assert { type: 'json' };
 import Circuit2 from '../artifacts/circuit_2.json' assert { type: 'json' };
-import Vk1 from '../artifacts/keys/circuit_1.vk.data.json' assert { type: 'json' };
-import Vk2 from '../artifacts/keys/circuit_2.vk.data.json' assert { type: 'json' };
 import type { FixedLengthArray } from './types/index.js';
 
 export const logger = createDebug('aztec:bb-bench');
 
 export const MOCK_MAX_COMMITMENTS_PER_TX = 4;
 
-function foreignCallHandler(): Promise<ForeignCallOutput[]> {
-  throw new Error('Unexpected foreign call');
-}
-
 export type u8 = string;
 
 export async function generateCircuit1(): Promise<[string, Uint8Array, InputValue]> {
   const program = new Noir(Circuit1);
-  const { witness, returnValue } = await program.execute(
-    {
-      x: '0x1',
-      y: '0x10',
-    },
-    foreignCallHandler,
-  );
+  const { witness, returnValue } = await program.execute({
+    x: '0x1',
+    y: '0x10',
+  });
   logger('generated circuit 1');
   return [Circuit1.bytecode, witness, returnValue];
 }
 
 export async function generateCircuit2(
-  proverOutput: ProverOutputForRecursion,
-  previousVk: string[],
+  proof: FixedLengthArray<string, 459>,
+  public_inputs: FixedLengthArray<string, 2>,
+  vk: FixedLengthArray<string, 128>,
```

Comment:
Magic numbers! I've been trying to root out things like this which make it very difficult to make minor changes to backend components like VK/proof size. Doesn't have to be fixed here but please try to avoid! (If you can't extract actual constants from the relevant constants file, at least name them equivalently in this file)

Reply:
Ah this was something that was already there in a difference place (`export type ProverOutputForRecursion` which is now deleted), but we can probably remove this alrogether